package.json

```diff
@@ -259,9 +259,9 @@
     "@lezer/common": "1.0.2",
     "@lezer/highlight": "1.1.3",
     "@lezer/lr": "1.3.3",
-    "@locker/near-membrane-dom": "^0.12.15",
-    "@locker/near-membrane-shared": "^0.12.15",
-    "@locker/near-membrane-shared-dom": "^0.12.15",
+    "@locker/near-membrane-dom": "0.13.3",
```

Comment:
Following on the conversation from the renovate PR... you can add it to the renovate ignore list [here](https://github.com/grafana/grafana/blob/main/.github/renovate.json5#L6-L19).

Reply:
I added it to the list as part of this PR